### PR TITLE
Ruijia update ug

### PR DESCRIPTION
**Pull request by matochichap**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -158,19 +158,26 @@ These are all the features in regard to a member in CMC.
...
+:information_source: **Notes about duplicate members:**
+
+Duplicate members are not allowed to be added into the list of members.
+
+An member is considered a duplicate if the member's name, phone number, email **or** telegram handle matches that of an existing member.
+
+This means that only a member with a unique name, phone number, email and telegram handle can be added into the list of members.
+</div>
+
+- `memberName`: Only alphabetical characters, @, () are allowed
```
Comment (jonasongg) [inline docs/UserGuide.md]: image

I think we wanted input parameters to be in callout boxes like in the picture above! Also might need to change the email format to match the actual regex validation of the code, as well as add this for tag and task

File: docs/UserGuide.md [hunk 2]
```diff
@@ -308,8 +313,39 @@ Tasks:
 
 ### 4.2 Applicant Features
 
+These are all the features in regard to an applicant in CMC.
+
 #### 4.2.1 Adding an applicant: `addApplicant` or `adda`
 
+Adds an applicant to the list of applicants in CMC.
+
+- Use `addApplicant` or `adda` to use add an applicant
```
Comment (jonasongg) [inline docs/UserGuide.md]: This line might not be necessary?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -308,8 +313,39 @@ Tasks:
...
+
+#### Usage:
```
Comment (jonasongg) [inline docs/UserGuide.md]: Same as above

File: docs/UserGuide.md [hunk 4]
```diff
@@ -158,19 +158,26 @@ These are all the features in regard to a member in CMC.
 Adds a member to the list of members in CMC.
 
 - Use `addMember` or `addm` to use add a member
-- Use `addApplicant` or `adda` to use add an applicant
 
 #### Usage:
```
Comment (jonasongg) [inline docs/UserGuide.md]: Think should change this to Format: for consistency?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -308,8 +313,39 @@ Tasks:
...
+:information_source: **Notes about duplicate applicants:**
+
+Duplicate applicants are not allowed to be added into the list of applicants.
+
+An applicant is considered a duplicate if the applicant's name **or** phone number matches that of an existing applicant.
+
+This means that only an applicant with a unique name and phone number can be added into the list of applicants.
+</div>
+
+- `applicantName`: Only alphabetical characters, @, () are allowed
```
Comment (jonasongg) [inline docs/UserGuide.md]: Same as above

File: docs/UserGuide.md [hunk 6]
```diff
@@ -308,8 +313,39 @@ Tasks:
...
+- `phoneNumber`: Only numbers are allowed
+
+#### Example of usage:
+
+`adda /name John Doe /phone 92345678`
```
Comment (jonasongg) [inline docs/UserGuide.md]: image

I think can follow this image, add bullet point and description of what this example does. Also might need to add the addApplicant.png image below this

File: docs/UserGuide.md (hunk 1 above)
Comment (matochichap) [reply docs/UserGuide.md]: Added issue for alfred to edit that

**Review by jonasongg — COMMENTED** [review]
LGTM, thanks for the changes!

**Pull request merged**